src/Arcus.WebApi.Hosting.AzureFunctions/Formatting/AzureFunctionsJsonFormattingMiddleware.cs

```diff
@@ -80,7 +80,7 @@ private static bool AcceptsJson(HttpRequestData request, ILogger logger)
         {
             if (request.Headers.TryGetValues("Accept", out IEnumerable<string> headerValues))
             {
-                if (headerValues.Any(value => value == "application/json" || value == "*/*"))
+                if (headerValues.Any(value => value?.Contains("application/json") is true || value?.Contains("*/*") is true))
```

Comment:
When I think about this, why do we by default restrict that clients can only understand json ? 
Because this is really what the accept header is used for.  The client indicates what kind of mime-type the client can accept.  This means that the server can use this information to decide in what format the request should be returned.
So, isn't it a bit weird that we enforce that this can only be json by default ?  I think we (Arcus) should not intervene here ?

In fact, it is weird that we accept 'json' or  everything ('*/*').

Reply:
This is actually an alternative to the input/output formaters in a Web API. There we also have restricted the input and output to only JSON. In Azure Functions we do not have that possibility, so we created it ourselves.
So, this is actually already there in Web API, and was also already in the in-process Azure Functions, we only now were able to do this in isolated Azure Functions.

Reply:
I just ask myself why Arcus needs to decide that.  But that's another discussion :)